packages/@aws-cdk/cli-lib/lib/cli.ts

```diff
@@ -0,0 +1,237 @@
+import * as core from '@aws-cdk/core';
+import { exec as runCli } from 'aws-cdk/lib';
```

Comment:
Minimal example to proof this works with the bundled package:
```sh
npm install aws-cdk 
node -e "const { cli } = require('aws-cdk/lib'); cli()"
```

`cli` is the existing export. I have no reason to believe `exec` would behave any different. Any further proof is very tricky within the constraints of the build setup (the bundled version of `aws-cdk` is only created at package time and as a tarball).

Reply:
Additionally confirmed this by taking the produced tarball from our test build pipeline and used it in an app.